microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -901,7 +901,13 @@ def verify_no_pre_exist_users(self, node: Node) -> None:
             r" \\\"root\\\".\';echo;sleep .*\"",
             re.M,
         )
-        current_user = node.tools[Whoami].get_username()
+        # For Bitnami images, the bitnami.service changes the uid of current user as
+        # 1000 which user 'bitnami' already has.
+        # From https://github.com/coreutils/coreutils/blob/master/src/whoami.c, whoami
+        # gets name from EUID which is 1000. Then the output is 'bitnami' rather than
+        # the current user. So modified to use remote_node._connection_info.username
+        remote_node = cast(RemoteNode, node)
+        current_user = remote_node._connection_info.username
```

Comment:
Another thing is, it needs to read user name from `connection_info`, not `_connection_info`. The `_connection_info` is a private member.